pkg/component/etcd/etcd/waiter.go

```diff
@@ -47,33 +47,33 @@ func (e *etcd) WaitCleanup(_ context.Context) error { return nil }
 
 // CheckEtcdObject checks if the given Etcd object was reconciled successfully.
 func CheckEtcdObject(obj client.Object) error {
-	etcd, ok := obj.(*druidv1alpha1.Etcd)
+	e, ok := obj.(*druidv1alpha1.Etcd)
```

Comment:
Please revert this rename, `etcd` seems more appropriate than `e`

Reply:
The name of the variable shadows `etcd` type defined at https://github.com/gardener/gardener/blob/master/pkg/component/etcd/etcd/etcd.go#L137 (since they share the same package hence the visibility). I think Shreyas changed the name to avoid shadowing which is generally recommended. If you have a strong opinion on renaming this then i can rename that.